Readme.txt

```diff
@@ -0,0 +1 @@
+this is a basic documentation for the sample code.
```

Comment:
This should have a capital at the start of a sentence

Reply:
Ok. I will update

Reply:
Resolve